storagenode/storagenodedb/database.go

```diff
@@ -56,27 +113,40 @@ func New(log *zap.Logger, config Config) (*DB, error) {
 	}
 	pieces := filestore.New(log, piecesDir)
 
-	infodb, err := newInfo(config.Info2)
+	dbs, err := boltdb.NewShared(config.Kademlia, kademlia.KademliaBucket, kademlia.NodeBucket, kademlia.AntechamberBucket)
 	if err != nil {
 		return nil, err
 	}
 
-	dbs, err := boltdb.NewShared(config.Kademlia, kademlia.KademliaBucket, kademlia.NodeBucket, kademlia.AntechamberBucket)
+	versionsPath := config.Info2
+	versionsDB, err := openDatabase(versionsPath)
```

Comment:
just curious- what does versionsdb do?

Reply:
This was already in place previously inside of `InfoDB` but `InfoDB` was one single database for all tables. This sets the table for a future PR to split SQLite3 tables into their own databases for performance purposes.

We already had a schema versions table that was encapsulated within `InfoDB`. We still need it to run migrations etc. So in my re-organizing of objects I needed to retain that functionality.